src/rpg_core/Graphics.js

```diff
@@ -376,14 +371,7 @@ Graphics.playVideo = function(src) {
     this._video.onerror = this._onVideoError.bind(this);
     this._video.onended = this._onVideoEnd.bind(this);
     this._video.load();
-
-    if (Utils.isMobileSafari()) {
```

Comment:
I replaced them with this._videoLoading rather than deleting them,
because RMMV waited until the video ends in mobileSafari only.
I apologize that this PR is hard to understand because it has multiple changes.
From now on I will send a simple one in the future.